packages/client/src/client.js

```diff
@@ -334,7 +334,7 @@ export class PercyClient {
     }
     let encodedContent = base64encode(content);
 
-    this.log.debug(`Uploading ${formatBytes(encodedContent.length)} resource: ${url}...`);
+    this.log.debug(`Uploading ${formatBytes(encodedContent.length)} resource for snapshotId: ${snapshotId}: ${url}...`);
```

Comment:
```suggestion
    this.log.debug(`Uploading ${formatBytes(encodedContent.length)} resource for snapshotId ${snapshotId} : ${url}...`);
```
avoiding double `:`

Reply:
Better option is to put build id and snapshot id in meta though. That way it will be automatically extracted and downloaded in snapshot logs

Reply:
Ideally update all logs in client to use meta which contains useful info like { build id, snapshot id, comparison id, url if applicable etc }